mangaki/mangaki/views.py

```diff
@@ -891,6 +892,51 @@ def fix_suggestion(request, suggestion_id):
     return render(request, 'fix/fix_suggestion.html', context)
 
 
+def nsfw_grid(request):
+    #FIXME: remove duplicate suggestions OR group by work ID
+    nsfw_suggestion_list = Suggestion.objects.select_related(
+            'work', 'user', 'work__category').prefetch_related(
+            'work__category', 'evidence_set__user').filter(
+            problem__in=['nsfw', 'n_nsfw'], is_checked=False).exclude(
+            evidence__isnull=False
```

Comment:
I'm not exactly sure why we should exclude suggestions where there is no evidence?
Isn't the moment where we can create the first evidence now?

Correct me if I'm wrong (I'm reviewing this in an airport at 01:00, sounds not a very good idea… :D).